test/test_overrides.py

```diff
@@ -29,6 +31,14 @@
 
 Tensor = torch.Tensor
 
+if os.getenv("ATEN_CPU_CAPABILITY") in ("default", "avx2"):
+    # This test is not supported on ARM
```

Comment:
@clee2000 Just came across this: The comment seems wrongly pasted from somewhere else.

I assume this test is supposed to be skipped when no GPU is available. So wouldn't checking TEST_CUDA be better?